### PR TITLE
Neutrinorpc: deprecate getblockhash

### PR DESCRIPTION
**Pull request by ErikEk**
Depreciates `neutrinorpc getblockhash`. It has been moved to `chainrpc` in 0.16. The lncli: `neutrino getblockhash` command has already been removed, but still accessible though the REST interface.

### CODE REVIEW

File: lnrpc/neutrinorpc/neutrino.yaml [hunk 1]
```diff
@@ -20,4 +20,5 @@ http:
     - selector: neutrinorpc.NeutrinoKit.GetCFilter
       get: "/v2/neutrino/cfilter/{hash}"
     - selector: neutrinorpc.NeutrinoKit.GetBlockHash
+    # deprecated
```
[the remark this answers is not in the document]
Comment (ErikEk) [reply lnrpc/neutrinorpc/neutrino.yaml:23]: Looks like we usually put them inside/underneath.
https://github.com/lightningnetwork/lnd/blob/3d53d6909a5428ce466884957e51e76febf919aa/lnrpc/routerrpc/router.yaml#L17

[the remark this answers is not in the document]
Comment (guggero) [reply lnrpc/neutrinorpc/neutrino.yaml]: You're right, but in that case the comment was placed there to show why there is no `get/post/delete` mapping below the selector. Here we have a mapping and we want to declare the whole entry as deprecated.

[the remark this answers is not in the document]
Comment (ErikEk) [reply lnrpc/neutrinorpc/neutrino.yaml]: @guggero Fixed.

[the remark this answers is not in the document]
Comment (ErikEk) [reply lnrpc/neutrinorpc/neutrino.yaml:22]: @guggero Fixed.

**Review by yyforyongyu — APPROVED** [review]

**Pull request merged**